src/isolated/FunctionsAuthorizationMiddleware.cs

```diff
@@ -83,34 +87,17 @@ public async Task Invoke(FunctionContext context, FunctionExecutionDelegate next
                 return;
             }
 
-            var authenticateResult = await _policyEvaluator.AuthenticateAsync(filter.Policy, httpContext);
-
-            var authenticateFeature = httpContext.Features.SetAuthenticationFeatures(authenticateResult);
+            var authenticateFeature = context.Features.Get<IAuthenticateResultFeature>();
 
-            // We also make the features available in the FunctionContext 
-            context.Features.Set<IAuthenticateResultFeature>(authenticateFeature);
-            context.Features.Set<IHttpAuthenticationFeature>(authenticateFeature);
-
-            if (filter.AllowAnonymous)
```

Comment:
Why is this check removed? We still need to allow for execution of an action regardless of authentication result if it's marked with `AllowAnonymous`